components/tools/travis-build

```diff
@@ -34,7 +34,7 @@ build_java()
 
 build_python()
 {
-    flake8 -v .
+    flake8 .
```

Comment:
Yes. Verbosity was useful to check what components were flake8'd but since we have now most of the Python code covered, this option results in more noise than necessary while trying to identify the cause of Travis failures (See commit message). With this change, flake8 failures should now render as https://travis-ci.org/openmicroscopy/openmicroscopy/jobs/48101273.
/cc @ximenesuk @joshmoore